src/app/public/src/modules/image/image.component.ts

```diff
@@ -8,4 +8,7 @@ import { Component, Input } from '@angular/core';
 export class StacheImageComponent {
   @Input()
   public imageSource: string;
+
+  @Input()
+  public imageAlt: string;
```

Comment:
Set the default to `''` so that we don't get:

<img width="350" alt="screen shot 2017-09-12 at 10 56 29 am" src="https://user-images.githubusercontent.com/12497062/30332731-1641d940-97a9-11e7-82a8-a84f5fa13f67.png">

Reply:
Ah, good call.